### DISCUSSION TITLE
Vs code error

### DISCUSSION THREAD

**Discussion by chaaritra21** [microsoft/vscode-cpptools#7290] [General]
[Running] cd "d:\Code\" && g++ demo.cpp -o demo && "d:\Code\"demo
c:/mingw/bin/../lib/gcc/mingw32/6.3.0/../../../libmingw32.a(main.o):(.text.startup+0xa0): undefined reference to `WinMain@16'
collect2.exe: error: ld returned 1 exit status

why this error coming again and again please help me out

**Comment by sean-mcmanus** [comment]
Did you define a `int main()` function? Is the mingw bin folder in your $PATH? We have some docs at https://code.visualstudio.com/docs/cpp/config-mingw .
